### PR TITLE
Cld 13798 add test set cookie

### PR DESCRIPTION
**Pull request by AlexeyCL**

### CODE REVIEW

File: src/test/specs/SetCookieSpec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  it('validate cookie name', () => {
+    assert.equal(retrievedCookiesArray[0].name, cookie.name, 'Incorrect cookie name');
+  });
+
+  it('validate cookie value', () => {
+    assert.equal(retrievedCookiesArray[0].value, cookie.value, 'Incorrect cookie value');
+  });
+
+  it('validate cookie with null domain value', () => {
+    // tslint:disable-next-line:no-null-keyword
```
Comment (FelixZilber) [inline src/test/specs/SetCookieSpec.ts]: remove

File: src/test/specs/SetCookieSpec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  it('validate cookie name', () => {
+    assert.equal(retrievedCookiesArray[0].name, cookie.name, 'Incorrect cookie name');
+  });
+
+  it('validate cookie value', () => {
+    assert.equal(retrievedCookiesArray[0].value, cookie.value, 'Incorrect cookie value');
+  });
+
+  it('validate cookie with null domain value', () => {
```
Comment (FelixZilber) [inline src/test/specs/SetCookieSpec.ts]: change test name

File: src/test/specs/SetCookieSpec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    // tslint:disable-next-line:no-null-keyword
+    BrowserUtils.setCookie(cookie, 'https://cloudinary.com/');
+    retrievedCookiesArray = browser.getCookies([cookie.name]);
+
+    assert.equal(retrievedCookiesArray[0].domain, TEST_PAGE_DOMAIN, 'Incorrect cookie domain');
```
Comment (FelixZilber) [inline src/test/specs/SetCookieSpec.ts]: add validation of current url to be the test page url

File: src/test/specs/SetCookieSpec.ts (hunk 1 above)
Comment (AlexeyCL) [reply src/test/specs/SetCookieSpec.ts]: done

File: src/test/specs/SetCookieSpec.ts (hunk 3 above)
Comment (AlexeyCL) [reply src/test/specs/SetCookieSpec.ts]: added

File: src/test/specs/SetCookieSpec.ts (hunk 2 above)
Comment (AlexeyCL) [reply src/test/specs/SetCookieSpec.ts]: changed

**Pull request merged**